crafters/nlp/PDFExtractorSegmenter/tests/test_pdf_flow.py

```diff
@@ -2,20 +2,28 @@
 import os
 
 
+from jina.flow import Flow
+import os
+
+
 def print_res(resp):
```

Comment:
rename it to `validate_text_fn`

Reply:
You think i should I still print something there or is not necessary?

Reply:
no, in a test there is no point in printing anything